lib/db.ts

```diff
@@ -1,18 +1,20 @@
-import { Db } from '../types'
+import { Db } from '../types';
+import * as firebaseDB from './dbs/firebase';
+import * as sqlDB from './dbs/mysql';
 
 const { DB_TYPE } = process.env;
 
 let db: Db;
 
 switch (DB_TYPE) {
     case 'firebase':
-        db = require('./dbs/firebase');
```

Comment:
this seemed to be creating issues with webpack5. was receiving an error, `WEBPACK_IMPORTED_MODULE_2__.default.setUser is not a function`